tests/Estranged.Lfs.Tests/Estranged.Lfs.Tests.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.8.0" />
```

Comment:
This is already pulled in via the azure project - no need to reference in the test project